File: solutions/beecrowd/1018/1018.py
bills = [100, 50, 20, 10, 5, 2, 1]

n = int(input())

print(n)

for bill in bills:
    print(f'{n // bill} nota(s) de R$ {bill},00')

    if n >= bill:
        n -= bill * (n // bill)

File: solutions/beecrowd/1018/1018.js
const input = require('fs').readFileSync('/dev/stdin', 'utf8')
const bills = [100, 50, 20, 10, 5, 2, 1]

let n = parseInt(input.trim())

console.log(n)

for (let i = 0; i < bills.length; i++) {
  console.log(`${Math.trunc(n / bills[i])} nota(s) de R$ ${bills[i]},00`)
  if (n >= bills[i]) {
    n -= bills[i] * Math.trunc(n / bills[i])
  }
}
